header.js

```diff
@@ -0,0 +1,27 @@
+export const header = `
+<header>
```

Comment:
If there's already a header tag in the html, you don't need to add it there because if you do there will be a header in a header tag.
You can delete line 2 (open header tag) and line 26 (closing header tag)